webpack.config.js

```diff
@@ -65,11 +65,28 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					'sass-loader',
+					"webpack-append?@import 'assets/stylesheets/shared/component';",
+				]
+			},
+			{
+				test: /\/client\/components\/vertical-nav.+index\.jsx$/,
```

Comment:
I'm not sure how will that work in the "general" case, should we create a 'generic' loader
that will do that instead of hard-coding it?

```js
{
   test: /\.scss/,
   loaders: [
'isomorphic-style-loader',
					'css-loader?camelCase=dashes&importLoaders=1',
 					'sass-loader',
 					"maybe-append-import-and-hoc",
   ]
```
? Or i missed something?